apps/backoffice-v2/src/pages/Entity/components/Case/hooks/useActions/useActionsLogic.tsx

```diff
@@ -24,10 +24,23 @@ export const ResubmissionReason = {
   FACE_IS_NOT_MATCHING: 'FACE_IS_NOT_MATCHING',
 } as const;
 
-export const useActions = ({ workflowId, fullName }: IUseActions) => {
-  const onSelectNextEntity = useSelectNextEntity();
+export const useActionsLogic = ({ workflowId, fullName }: IUseActions) => {
+  // <Current entity>
```

Comment:
Exploring two new conventions I will really appreciate feedback and will revert if decided against the idea. :pray: ;

1. Opening a group of related logic with `<[Subject]>` and closing with `</[Subject]>`. 
2. A suffix of `Logic` for hooks that just encapsulate state and business logic of components.